egs/madcat_ar/v1/local/extract_features.sh

```diff
@@ -36,7 +37,7 @@ utils/split_scp.pl $scp $split_scps || exit 1;
 $cmd JOB=1:$nj $logdir/extract_features.JOB.log \
   local/make_features.py $logdir/images.JOB.scp \
```

Comment:
Would it be possible to use the shared version of make_features.py here (I guess it's located at image/ocr/) w/o making too many changes?

Reply:
Ok, thanks. I will try to use the shared version of make_features.